proto/iamanager/v5/iamanager.proto

```diff
@@ -7,7 +7,7 @@ import "common/v1/common.proto";
 
 service IAMPublicService {
     rpc GetNodeInfo(google.protobuf.Empty) returns (NodeInfo) {}
-    rpc GetCert(GetCertRequest) returns (GetCertResponse) {}
```

Comment:
We should keep `GetCert` method IMHO. We work with storages in two ways:
1. Providing cert serial and issuer to get URL for specific certificate. Used for offline cert. And `GetCert` method will be used for it.
2. Get newest cert for specific storage. In this `SubscribeCert` method will be suitable.

Reply:
fixed